packages/rnv/pluginTemplates/renative.plugins.json

```diff
@@ -1523,6 +1523,141 @@
                 "package": "com.wscodelabs.callLogs.CallLogPackage"
             }
         },
+        "@react-native-firebase/app": {
+            "version": "6.4.1",
+            "ios": {
+                "isStatic": true,
+                "podName": "RNFBApp"
+            },
+            "android": {
+                "package": "io.invertase.firebase.app.ReactNativeFirebaseAppPackage",
+                "projectName": "@react-native-firebase_app",
+                "build.gradle": {
+                    "buildscript": {
+                        "dependencies": [
+                            "classpath 'com.google.gms:google-services:4.2.0'"
+                        ]
+                    }
+                },
+                "app/build.gradle": {
+                    "apply": ["plugin: 'com.google.gms.google-services'"]
+                }
+            },
+            "androidtv": {
+                "package": "io.invertase.firebase.app.ReactNativeFirebaseAppPackage",
+                "projectName": "@react-native-firebase_app",
+                "build.gradle": {
+                    "buildscript": {
+                        "dependencies": [
+                            "classpath 'com.google.gms:google-services:4.2.0'"
+                        ]
+                    }
+                },
+                "app/build.gradle": {
+                    "apply": ["plugin: 'com.google.gms.google-services'"]
+                }
+            }
+        },
+        "@react-native-firebase/analytics": {
+            "version": "6.4.1",
+            "dependsOn": ["@react-native-firebase/app"],
+            "ios": {
+                "isStatic": true,
+                "podName": "RNFBAnalytics",
+                "xcodeproj": {
+                    "resourceFiles": ["RNVApp/GoogleService-Info.plist"]
+                },
+                "appDelegateImports": ["Firebase"],
+                "appDelegateMethods": {
+                    "application": {
+                        "didFinishLaunchingWithOptions": [
+                            "FirebaseApp.configure()"
+                        ]
+                    }
+                }
+            },
+            "android": {
+                "package": "io.invertase.firebase.analytics.ReactNativeFirebaseAnalyticsPackage",
+                "projectName": "@react-native-firebase_analytics"
+            },
+            "androidtv": {
+                "package": "io.invertase.firebase.analytics.ReactNativeFirebaseAnalyticsPackage",
+                "projectName": "@react-native-firebase_analytics"
+            }
+        },
+        "@react-native-firebase/crashlytics": {
+            "version": "6.4.1",
+            "dependsOn": ["@react-native-firebase/crashlytics"],
```

Comment:
self reference?

Reply:
fixed - changed to analytics (which guarantees the proper initialisation)